docs/reference/migration/migrate_7_0/api.asciidoc

```diff
@@ -79,3 +79,11 @@ the only behavior in 8.0.0, this parameter is deprecated in 7.0.0 for removal in
 ==== The deprecated stored script contexts have now been removed
 When putting stored scripts, support for storing them with the deprecated `template` context or without a context is
 now removed. Scripts must be stored using the `script` context as mentioned in the documentation.
+
+==== Get Aliases API limitations when xpack security is enabled removed
+
+When xpack security is enabled, the Get Aliases API behaviour and response
+code returned are now aligned to those of vanilla Elasticsearch. Previously
```

Comment:
In my opinion "vanilla" might be confusing, since X-Pack is included in Elasticsearch by default now.  Maybe something like this might be clearer?: "The behavior and response codes of the get aliases API no longer vary depending on whether {security} is enabled. Previously..."